llm_demo/evaluation/eval_golden.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+goldens = [
+    {
+        "Search Airport Tool": [
```

Comment:
Curious to the structure -- why is this list -> object {key: object{} }?

Wouldn't it be better to just use an object at the higher level with key -> object?

Reply:
Each evaluation data object contains of "query, tool call, output etc.". 

And since I am trying to categorize them into different categories (e.g. we'll provide 2 test data for 'Search Airport Tool' so that we can test out 2 different queries that trigger that tool. If we want to add more queries in the future we can just add it to the category's list.), that's why we have list of evaluation data based on the category (a.k.a. key).